demo/index.tsx

```diff
@@ -27,6 +27,12 @@ declare global {
   }
 }
 
+// TODO: we should import this from CK types package @types/cryptkeeper
+interface SelectedIdentity {
+  commitment: string;
+  web2Provider?: string;
```

Comment:
@iskdrews
As it will be for interep groups, perhaps we could try to follow the parameters names used by interep?
No big deal but it'll be a bit simpler to integrate all the pieces together.
interep just call this `provider`
https://docs.interep.link/api/#apiv1groups